### PR TITLE
Add django-filer

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/django-filer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
```
Comment (isuruf) [inline recipes/django-filer/meta.yaml]: This differs from what's in setup.py

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge if everything looks okay.

**Comment by scopatz** [comment]
@sannykr please address @isuruf's comment, thanks!

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge if everything looks okay.

File: recipes/django-filer/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipes/django-filer/meta.yaml]: @sannykr, can you please address this issue before asking for another review

Comment (sannykr) [reply recipes/django-filer/meta.yaml]: @isuruf: Sorry about that. I just updated the django-version.

**Pull request merged**